packages/framework/client-logger/fluid-telemetry/src/factory/index.ts

```diff
@@ -9,30 +9,30 @@ import { type ITelemetryConsumer } from "../common/index.js";
 import type { IFluidContainer } from "@fluidframework/fluid-static";
 
 /**
- * Configuration object for subscribing to {@link IExternalTelemetry} and consuming said telemetry via one or more {@link ITelemetryConsumer}
+ * Configuration object for subscribing to {@link @fluidframework/fluid-telemetry#IFluidTelemetry} and consuming said telemetry via one or more {@link ITelemetryConsumer}
  *
  * @beta
  */
 export interface TelemetryConfig {
 	/**
-	 * The container whose events should be monitored, transformed into external telemetry, and sent to a {@link ITelemetryConsumer}.
+	 * The container whose events should be monitored, transformed into Fluid telemetry, and sent to a {@link ITelemetryConsumer}.
 	 */
 	container: IFluidContainer;
 	/**
-	 * Unique identifier for a container, stable across creation and load.
-	 * I.e. different clients loading the same container (or the same client loading the container two separate times)
-	 * will agree on this value.
+	 * Unique identifier for the passed in container, i.e. the return value of a call
+	 * to {@link @fluidframework/fluid-static#IFluidContainer.attach | `IFluidContainer.attach()`} when creating a new
+	 * Fluid container, or the id used to load a pre-existing one.
 	 */
 	containerId: string;
 	/**
-	 * Conusmers take incoming produced {@link IExternalTelemetry} and do something of your choice with it.
+	 * Conusmers take incoming produced {@link IFluidTelemetry} and do something of your choice with it.
```

Comment:
I think this works to link to IFluidTelemetry in TSDoc, so line 12 could be the same, right? Or do we need the full package "path" for it?

Reply:
I'm not sure so I updated this to the full package path